document-picture-in-picture/main.js

```diff
@@ -0,0 +1,90 @@
+
+const videoPlayer = document.getElementById("player");
+const playerContainer = document.getElementById("container");
+let togglePipButton;
+let pipActive = false;
+let pipWindow;
+
+const inPIPMessage = document.createElement("p");
```

Comment:
You may want to move this in HTML and simply querySelector it in JS. WDYT?

Reply:
Yeah, I guess so. The code is about as complex, but it serves to cut down on a bit of DOM scripting. I've put it in the HTML, and am instead showing/hiding it by manipulating the `display` property in JS.